myosuite/envs/myo/myochallenge/chasetag_v0.py

```diff
@@ -374,6 +386,176 @@ def ncol(self):
         return self.hfield.ncol
 
 
+class RepellerChallengeOpponent(ChallengeOpponent):
+    # Repeller parameters
+    DIST_INFLUENCE = 3.5 # Distance of influence by the repeller
+    ETA = 20.0 # Scaling factor
+    MIN_SPAWN_DIST = 1.5
+    BOUND_RESOLUTIONS = [-8.7, 8.7, 25]
+
+    def __init__(self,
+                 sim,
+                 rng,
+                 probabilities: tuple[float],
+                 min_spawn_distance: float,
+                 chase_vel_range: tuple[float],
+                 random_vel_range: tuple[float],
+                 repeller_vel_range: tuple[float],
+        ):
+        self.dt = 0.01
```

Comment:
Is there a reason for this to be 10ms. Should it be taken form the env's dt?

Reply:
will change to take it from the env